File: sbt-conductr-tester/lagom-java-bundle/lagom-service-impl/src/main/java/impl/FooServiceImpl.java
package impl;

import akka.NotUsed;
import api.FooService;
import com.lightbend.lagom.javadsl.api.ServiceCall;

import java.util.concurrent.CompletableFuture;

public class FooServiceImpl implements FooService {

  @Override
  public ServiceCall<NotUsed, String> foo() {
    return request -> CompletableFuture.completedFuture("hardcoded-foo-response");
  }
}

File: src/sbt-test/public/lagom-conductr-plugin-legacy-scala-service/lagom-service-impl/src/main/scala/impl/FooServiceImpl.scala
package impl

import api.FooService
import com.lightbend.lagom.scaladsl.api.ServiceCall

import scala.concurrent.Future

class FooServiceImpl extends FooService {
  override def foo = ServiceCall { _ =>
    Future.successful("foo-response-hardcoded in FooServiceImpl")
  }
}
